ctms/schemas/newsletter.py

```diff
@@ -43,3 +45,23 @@ class Config:
 # No need to change anything, just extend if you want to
 NewsletterInSchema = NewsletterBase
 NewsletterSchema = NewsletterBase
+
+
+class NewsletterTableSchema(NewsletterBase):
+    email_id: UUID4 = Field(
+        description=EMAIL_ID_DESCRIPTION,
+        example=EMAIL_ID_EXAMPLE,
+    )
+    create_timestamp: datetime = Field(
+        default=None,
+        description="Newsletter subscription data creation timestamp",
+        example="2020-12-05T19:21:50.908000+00:00",
+    )
+    update_timestamp: datetime = Field(
+        default=None,
```

Comment:
'None' defaults for these?

Reply:
Oh yeah, we should remove these defaults although I think they do nothing here technically. good catch